.github/workflows/check-quay-image.yml

```
@@ -1,18 +1,21 @@
 name: Check Quay Image
 
 on:
-  push:
+  check_suite:
+    types:
+      - completed
     branches:
```

actionlint (1.7.12)

[error] "branches" filter is not available for check_suite event. it is only for merge_group, push, pull_request, pull_request_target, workflow_run events

(events)